packages/jellyfish-api-core/__tests__/category/oracle/appointOracle.test.ts

```diff
@@ -23,10 +23,26 @@ describe('Oracle', () => {
       { currency: 'EUR', token: 'APPLE' }
     ]
 
-    const data = await client.oracle.appointOracle(await container.getNewAddress(), priceFeeds, { weightage: 1 })
+    const txid = await client.oracle.appointOracle(await container.getNewAddress(), priceFeeds, { weightage: 1 })
 
-    expect(typeof data).toStrictEqual('string')
-    expect(data.length).toStrictEqual(64)
+    expect(typeof txid).toStrictEqual('string')
+    expect(txid.length).toStrictEqual(64)
+
+    await container.generate(1)
+
+    const result = await container.call('getoracledata', [txid])
+    expect(result).toStrictEqual(
+      {
+        weightage: 1,
+        oracleid: txid,
+        address: expect.any(String), // This is an oracle address, not the address in line 26
```

Comment:
I assume this is an oracle address. Kindly tell me if my assumption is wrong.

Reply:
This check is sufficient for now, as the getoracledata rpc call is returning the script hex instead of the address

Reply:
Ok, then I remove the wrong comment